setup.md

```diff
@@ -26,89 +27,96 @@ title: Setup
 
 
 > ## Software
-> [Python](http://python.org) is a popular language for
-> scientific computing, and great for general-purpose programming as
-> well.  Installing all of its scientific packages individually can be
-> a bit difficult, so we recommend an all-in-one installer.
+> [**Python**](http://python.org) es un lenguaje popular para
+> la computación científica y también es ideal para la programación de propósito general.
+> La instalación de todos los paquetes científicos individualmente puede ser
+> un poco difícil, por lo que recomendamos un instalador todo en uno.
```

Comment:
Lines 32 and 33 creo q sería buena idea moverlas abajo antes de mencionar conda

Reply:
ese todo en uno no suena bien.
De aquí saquí una sugerencia https://www.tutorialpython.com/instalar-python/
Anaconda es una distribución de Python que incluye muchas herramientas y paquetes.